images/linux/scripts/tests/PowerShellModules.Tests.ps1

```diff
@@ -44,9 +43,21 @@ Describe "AzureModules" {
                         $env:PSModulePath = "${modulePath}:${env:PSModulePath}"
                         Import-Module -Name $moduleName
                         (Get-Module -Name $moduleName).Version.ToString()
+                    }
+                    
+                    if (Test-IsUbuntu18) {
```

Comment:
maybe follow this approach for all Ubuntus'?

Reply:
Agreed. If it doesn't change test reliability, then we can do this for all Ubuntu images and avoid conditions

Reply:
Thx. Fixed.